### ISSUE TITLE
Optionally add tags to notifications

### ISSUE THREAD

**Issue by michaelbarone** [louislam/uptime-kuma#2730]
### ⚠️ Please verify that this feature request has NOT been suggested before.

- [X] I checked and didn't find similar feature request

### 🏷️ Feature Request Type

New Notification, Other

### 🔖 Feature description

modification to the existing notifications framework.  adding the option to include tags (maybe per tag, or all tags would be fine) in the notifications that go out.

Currently, we are monitoring 2 systems.  API (production) and API (development).  the (tags) for each monitor are not included in the notification resulting in this:

Uptime Kuma Alert
Message
[API] [:red_circle: Down] Request failed with status code 502

Unfortunately, it doesnt tell me which environment is down, without changing the monitor names.  but the full monitor names show up on the status pages, and i dont really want to append -Production or -Development to all of my monitor names for the public status page, and currently only show the tags on our internal only status page.

### ✔️ Solution

Allow an option when configuring notifications to include the tags in the message.  They can be on their own line, comma separated.

We use the Slack notifications, if you want to test with this one before expanding to the other services :)

I know some notification services may run into length issues due to this request and might need to be addressed or omitted from this request.

### ❓ Alternatives

Adding the environment suffix to each monitor name so its included in the notification text.  I was hoping to leverage tags in this way without needing to make the monitor names longer than they need to be.

Also, I dont really want/need the environment tags/ or names on the public facing status page, but they are helpful for our internal facing status page.  That is why i started leveraging the tags i the first place, and it works well for that.

**Comment by titanventura** [comment]
would like to work on this !

**Comment by purplespider** [comment]
Would love this too. I have dozens of sites across multiple different servers on multiple different hosts, and while tags are the ideal place to store details of the server and host, I'd really like it if they could be displayed in the notifications so that I'd know instantly which server was down.

**Comment by ldubrois** [comment]
Just discovered Kuma ; great tool. 
We have several environment across the world. Having the tags in the Teams notification could be a good way to identify quickly the problem's location.

**Comment by bengivre** [comment]
I have been wanting to see this possibility since tags exist 👍

**Comment by CommanderStorm** [comment]
here is our contribution guide: https://github.com/louislam/uptime-kuma/blob/aa676150eb4cc803fe3de10f186a083f44a1c658/CONTRIBUTING.md

**Comment by JZ-SmartThings** [comment]
Please consider the Alternative method in the OP. Here's my simple use case and unfortunately changing Tags is way more work than pre-pending some unique identifier.

In Uptime Kuma backups it says to backup the entire docker volume folder instead of using the backup feature. Then I take that docker folder onto my test environment. When I do that, the same exact messages for failures are sent to me in Telegram. There is no way to make them unique so that I can tell the difference between Production and Test environments that run the same set of containers for me. I would love for NOTIFICATION settings (rather than tags) to have some prepend or append feature where I can add "T-" for test or "P-" for production and only do that change once after copying my docker folder. Having tags means I have to change 27 items in my case and hence the reason why adding this at the Notification config level.

Thank you very much for this AMAZING software! Way better than anything I've tried so I'm thoroughly impressed.

**Comment by syberx** [comment]
yes please add mor option to notification text. I need Tags in my Push Notifications
